src/components/editor/Bindings/Selector.tsx

```diff
@@ -24,9 +65,20 @@ const initialState = {
     },
 };
 
-function BindingSelector() {
+const typographyTruncation: TypographyProps = {
+    noWrap: true,
+    sx: {
+        minWidth: 0,
+    },
+};
```

Comment:
This is how words are being truncated in this instance, yes. If you would like me to move it to the theme, that is fine. Whether this will be useful elsewhere is impossible to determine at this point.